iroha_data_model/src/lib.rs

```diff
@@ -969,6 +1011,23 @@ pub mod asset {
         pub const fn new_store(id: DefinitionId) -> Self {
             AssetDefinition::new(id, AssetValueType::Store)
         }
+
+        /// Checks the length of the id is in a valid range
+        ///
+        /// # Errors
+        /// Fails if limit check fails
+        pub fn validate_len(&self, range: impl Into<RangeInclusive<usize>>) -> Result<()> {
+            let range = range.into();
+            if range.contains(&self.id.name.len()) {
```

Comment:
It should be noted that by checking this way we check the length in bytes, not in graphemes see [len docs](https://doc.rust-lang.org/std/string/struct.String.html#method.len) for more on this. If this is intentional, then please specify here in docs and in general config docs that we check string length in bytes.

Reply:
Added details in docs